build-system/tasks/e2e/selenium-webdriver-controller.js

```diff
@@ -590,6 +609,15 @@ class SeleniumWebDriverController {
     fs.writeFile(path, imageString, 'base64', function() {});
   }
 
+  /**
+   * @param {!ElementHandle} handle
+   * @return {!Promise}
+   * @override
+   */
+  async switchToWindow(handle) {
+    await this.driver.switchTo().window(handle.getElement());
```

Comment:
`this.driver.switchTo().window()` expects a string parameter, see https://github.com/SeleniumHQ/selenium/blob/master/javascript/node/selenium-webdriver/lib/webdriver.js#L1687